db/migrate/20190325023714_create_available_hours.rb

```diff
@@ -0,0 +1,12 @@
+class CreateAvailableHours < ActiveRecord::Migration[5.2]
+  def change
+    create_table :available_hours do |t|
+      t.time :start_time
+      t.time :end_time
+      t.string :days_of_week
```

Comment:
Preferiría cambiar esto

Reply:
enum para day of week? o q
`enum string: [:monday, :tuesday, :wednesday, :thursday, :friday, :saturday, :sunday ]`

Reply:
sii y nada más cambiar a singular el nombre